src/Tribe/Importer/Encoding.php

```diff
@@ -0,0 +1,338 @@
+<?php
+/*
+Copyright (c) 2008 Sebasti�n Grignoli
```

Comment:
Since this is external code, it should be placed in `/vendor/ForceUTF8`. Perhaps with the following class name: `ForceUTF8__Encoding`.